packages/leafygreen-provider/src/ModalPopoverContext/ModalPopoverContext.types.ts

```diff
@@ -0,0 +1,16 @@
+export interface ModalPopoverContextType {
+  /**
+   * Whether the most immediate popover ancestor is open
+   */
+  isPopoverOpen: boolean;
+
+  /**
+   * Sets the internal state
+   * @internal
+   */
+  setIsPopoverOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export interface ModalPopoverProviderProps {
```

Comment:
`PropsWithChildren<{}>` ? Could also just inline this type in the Provider component